University-advisor/Geocoding.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Device.Location;
+using System.Globalization;
+using University_advisor.Models;
+using Newtonsoft.Json;
+
+namespace University_advisor
+{
+    class Geocoding
+    {
+        string apiKey = "5e66cc9d64db23";
+        public double ReturnDistance(string urlA, string urlB)
+        {
+            var jsonA = new ApiCalls().GetLocationJson(urlA);
+            var jsonB = new ApiCalls().GetLocationJson(urlB);
+
+            var locationDataA = JsonConvert.DeserializeObject<List<LocationData>>(jsonA);
+            var locationDataB = JsonConvert.DeserializeObject<List<LocationData>>(jsonB);
+
+            //Using first member of array because there can be multiple returned locations
+            var locationA = new Geocoding().GetCoordinates(locationDataA[0]);
+            var locationB = new Geocoding().GetCoordinates(locationDataB[0]);
+
+            var distance = new Geocoding().GetDistance(locationA.Item1, locationA.Item2, locationB.Item1, locationB.Item2);
+            //Console.WriteLine(distance);
+            //Console.ReadLine();
+
+            return distance;
+        }
+
+        public (double, double) GetCoordinates(LocationData address)
```

Comment:
Not a mistake but I am wondering how this actually works? Method returns two double results, how do you select which one to choose when calling it?

Reply:
With this expression you receive tuple as returned object

Reply:
Great feature, thank you for explaining it.